app/root.tsx

```diff
@@ -122,13 +122,14 @@ type LoaderData = Awaited<ReturnType<typeof loaderCallback>>;
 
 const App = () => {
   const data = useLoaderData<LoaderData>();
+  const isDev = process.env.NODE_ENV === "development";
   return (
     <html lang="en" className="h-full">
       <head>
         <Meta />
         <Links />
       </head>
-      <body className="h-full">
+      <body className="h-full" suppressHydrationWarning={isDev}>
```

Comment:
hmmm - why did we add this?

Reply:
To get rid of this error on load.  I probably should have actually looked into it instead of suppressed it, but I just wanted to get work done.

[Extra attributes from the server](https://www.loom.com/share/f205f7ac9bf248f3b550129829ca7d9e)

nixed.